eks-anywhere-common/Testers/Sysdig/sysdig-test-cronjob.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: sysdig-agent-test
+  namespace: sysdig
+spec:
+  schedule: "0 1 * * *"
+  backoffLimit: 1
```

Comment:
spec.backoffLimit will need to be moved down to spec.jobTemplate.spec.backoffLimit to be valid for the Cronjob api, please update or remove

Reply:
Good catch, this one should be done now, thanks.